taker/src/routes.rs

```diff
@@ -60,6 +61,7 @@ pub async fn feed(
     rx: &State<Feeds>,
     rx_wallet: &State<watch::Receiver<Option<WalletInfo>>>,
     rx_maker_status: &State<watch::Receiver<ConnectionStatus>>,
+    rx_maker_identity: &State<watch::Receiver<Option<identify::PeerInfo>>>,
```

Comment:
🙃 I get confused with the mixed terminology between peer info, identity and identify. I think the chosen name `PeerInfo` is good enough, so it'd be good to be consistent.

```suggestion
    rx_maker_peer_info: &State<watch::Receiver<Option<identify::PeerInfo>>>,
```